discord/utils.py

```diff
@@ -1164,3 +1164,35 @@ def check(item: Any) -> bool:
         return iter(itertools.islice(gen, 25))
 
     return autocomplete_callback
+
+
+def filter_params(params, **kwargs):
+    """A helper function to filter out and replace certain keyword parameters
+
+    Parameters
+    -----------
+    params: Dict[str, Any]
+        The initial parameters to filter.
+    **kwargs: Dict[str, Optional[str]]
+        Key to value pairs where the key's contents would be moved to the
+        value, or if the value is None, remove key's contents (see code example).
+
+    Example
+    -------
+    .. code-block:: python3
+
+        >>> params = {"param1": 12, "param2": 13}
+        >>> filter_params(params, param1="param3", param2=None)
+        {'param3': 12}
+        # values of 'param1' is moved to 'param3'
+        # and values of 'param2' are completely removed.
+
+    """
+    for param, nparam in kwargs.items():
```

Comment:
`nparam` is a confusing variable name. `param_value` sounds better

Reply:
`nparam` was supposed to be the new param so I'll name it that